### DISCUSSION TITLE
Social Media Marketing, but for GitHub and programming

### DISCUSSION THREAD

**Discussion by OperatorHiKE** [community/community#111199] [Other Feature Feedback, Questions, & Ideas]
I've been asked to increase popularity on the GitHub account, and it seems similar to the SMM job. Do you have any strategy and ideas on how to perform well?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @OperatorHiKE , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
